README.md

```diff
@@ -19,4 +19,17 @@ plugins:
 - "@semantic-release/commit-analyzer"
 - "@semantic-release/release-notes-generator"
 - "@semantic-release/git"
+```
+
+Example using the success command hook :
+```yaml
+# .releaserc
+branches: ["master", "next"]
+plugins:
+  - "@semantic-release/changelog"
+  - "@semantic-release/commit-analyzer"
+  - "@semantic-release/git"
+  - "@semantic-release/release-notes-generator"
+  - - "@semantic-release/exec"
+    - successCmd: git push --mirror git@github.com:Adaptavist/example-repo.git
```

Comment:
Might be worth replacing the org name in that example. E.G _git@github.com:ExampleOrg/example-repo.git_

Reply:
yeah fair